packages/twilio-run/bin/upgrade-config.js

```
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+
+// Migration script to transform .twilio-functions files into the appropriate
+// .twilioserverlessrc and .twiliodeployinfo files
+
+const path = require('path');
+const inquirer = require('inquirer');
+const { fileExists, readFile, writeFile } = require('../dist/utils/fs');
+const { writeDefaultConfigFile } = require('../dist/templating/defaultConfig');
+const { readLocalEnvFile } = require('../dist/config/utils/env');
+const rimraf = require('rimraf');
+
+async function run() {
+  const oldConfigPath = path.resolve(
+    process.cwd(),
+    process.argv[2] || '.twilio-functions'
+  );
+
+  if (!fileExists(oldConfigPath)) {
+    console.error('Could not find an old config file at "%s"', oldConfigPath);
+    return 1;
+  }
+
+  const oldConfigContent = JSON.parse(await readFile(oldConfigPath, 'utf8'));
+
+  if (!oldConfigContent) {
```

Won't JSON parse throw an error rather than return `null`/`undefined`?

---

Good point! I'll get that fixed